### PR TITLE
Update en.json with new language strings

### PR DESCRIPTION
**Pull request by Patrick-Erichsen**
Signed-off-by: Patrick Erichsen <patrick.a.erichsen@gmail.com>

#### Description:

Updates `en.json` with new language strings and removes the `authorities_title` header from `ChooseProvider.js` as per @driedgrape 

#### Linked issues:

N/A

#### Screenshots:

Screen Shot 2020-05-09 at 9 16 05 PM

#### How to test:

Validate that the language string updates are correct.

### CODE REVIEW

File: app/views/ChooseProvider.js [hunk 1]
```diff
@@ -232,9 +232,6 @@ class ChooseProviderScreen extends Component {
           title={languages.t('label.choose_provider_title')}
           onBackPress={this.backToMain.bind(this)}>
           <View style={styles.main}>
-            <Typography style={styles.headerTitle} use={'headline2'}>
-              {languages.t('label.authorities_title')}
```
Comment (tstirrat) [inline app/views/ChooseProvider.js:236]: If this is the only place, remove the key from `en.json`

Comment (Patrick-Erichsen) [reply app/views/ChooseProvider.js:236]: Good catch.

**Comment by tstirrat** [comment]
Are these ready to merge? or still waiting.. if waiting, move to the other column.

**Comment by tstirrat** [comment]
These changes seem stale.. closing for now

**Pull request closed without merging**